packages/protocol/contracts/governance/LockedGold.sol

```diff
@@ -277,6 +277,11 @@ contract LockedGold is
     return (values, timestamps);
   }
 
+  /**
+   * @notice Returns the number of pending withdrawals from unlocked gold for an account.
+   * @param account The address of the account.
+   * @return total number of pending withdrawals
```

Comment:
Wording: I think this should be worded in terms of "amount to withdraw" rather than "number of withdrawals" - this is summing an amount of CELO I believe.

Reply:
right. I misunderstood the function